storage/docs/containers-storage-image.md

```diff
@@ -11,8 +11,9 @@ Retrieve information about an image: its ID, any names it has, and the ID of
 its top layer.
 
 ## EXAMPLE
-**containers-storage image 49bff34e4baf9378c01733d02276a731a4c4771ebeab305020c5303679f88bb8**
-**containers-storage image my-favorite-image**
+
+    containers-storage image 49bff34e4baf9378c01733d02276a731a4c4771ebeab305020c5303679f88bb8
+    containers-storage image my-favorite-image
```

Comment:
At least https://github.com/containers/podman/blob/main/docs/MANPAGE_SYNTAX.md suggests separate blocks for each example.

Reply:
done